### PR TITLE
CLOUDP-70027:  mongocli atlas integration(s) list|l

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-70027

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

```
./bin/mongocli atlas integrations list
TYPE
NEW_RELIC
OPS_GENIE
```

```
./bin/mongocli atlas integrations list -o json
{
  "links": [
    {
      "rel": "self",
      "href": "https://cloud-dev.mongodb.com/api/atlas/v1.0/groups/5e4e593f70dfbf1010295836/integrations?pageNum=1\u0026itemsPerPage=100"
    }
  ],
  "results": [
    {
      "type": "NEW_RELIC",
      "licenseKey": "eu01xx5f87268ff5028e69a67f3XXXXXXXX",
      "accountId": "2857404",
      "writeToken": "NRII-zpwCnhrDyUojdSJOk2XXXXXXXXXXX",
      "readToken": "NRIQ-MHCu7bPq2xvrrpVvybtXXXXXXXXXX"
    },
    {
      "type": "OPS_GENIE",
      "apiKey": "9255b505-9c55-45a4-893c-XXXXXXX",
      "region": "EU"
    }
  ],
  "totalCount": 2
}
```

### CODE REVIEW

**Comment by andreaangiolillo** [comment]
@gssbzn Happy to discuss how to make the output better

**Review by colm-quinn — DISMISSED** [review]
LGTM from a code perspective - probably should reach out to @gssbzn directly for opinions on the output that you are requesting.

File: internal/cli/atlas/integrations/list.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	r, err := opts.store.Integrations(opts.ConfigProjectID())
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+// mongocli atlas integrations(s) list --projectId projectId
+func ListBuilder() *cobra.Command {
```
Comment (gssbzn) [inline internal/cli/atlas/integrations/list.go:51]: can we add a test for this builder with the new CmdValidator?

**Review by gssbzn — COMMENTED** [review]
One small comment that we should start testing our builders

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**